src/PooledArrays.jl

```diff
@@ -132,19 +153,18 @@ function PooledArray{T}(d::AbstractArray, r::Type{R}) where {T,R<:Integer}
     end
 
     # Assertions are needed since _label is not type stable
-    PooledArray(RefArray(refs::Vector{R}), invpool::Dict{T,R}, pool)
+    return PooledArray(RefArray(refs::Vector{R}), invpool::Dict{T,R}, pool, Threads.Atomic())
```

Comment:
This is enough thanks to the default value, right?
```suggestion
    return PooledArray(RefArray(refs::Vector{R}), invpool::Dict{T,R}, pool)
```